package.json

```diff
@@ -47,15 +48,17 @@
   },
   "scripts": {
     "clean": "shx rm -rf lib esm bundles",
+    "coverage": "npm run test --coverage", 
     "test": "jest",
-    "test:ci": "jest --coverage && cat ./reports/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "coverage": "jest --coverage",
+    "test:ci": "npm run test --coverage && cat ./reports/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "test:format": "prettier --config ./config/.prettierrc --write \"{src,test}/**/*.{ts,tsx}\"",
     "docs": "cross-env NODE_ENV=development webpack-dev-server --config webpack.docs.js --env.dev",
     "deploy": "cross-env NODE_ENV=production webpack --config webpack.docs.js --env.prod",
     "build": "npm run clean && tsc && tsc -m es6 --outDir esm --removeComments false && cross-env NODE_ENV=production webpack",
     "watch:lib": "watch \"npm run deploy\" src/ --wait=2000 --ignoreUnreadable",
     "watch:docs": "watch \"npm run docs\" docs/ --wait=2000 --ignoreUnreadable",
     "watch:all": "parallelshell \"npm run watch:lib\" \"npm run watch:docs\"",
+    "pretest": "npm run test:format",
```

Comment:
Maybe we can use [husky](https://github.com/typicode/husky) to ensure that we run prettier pre-commit, what do you think?

Reply:
Yup, we can.  I'll add in this PR señor :D